rackcity/models/it_instance.py

```diff
@@ -22,3 +35,11 @@ class ITInstance(models.Model):
     class Meta:
         ordering = ['hostname']
         verbose_name = 'instance'
+
+    def save(self, *args, **kwargs):
+        try:
+            validate_hostname(self.hostname)
+        except ValidationError as valid_error:
+            raise valid_error
```

Comment:
Catching the error and raising it is a lil redundant but I don't hate it for readability

Reply:
feel that.